packages/sdk/src/depositStorage/depositStorage.test.ts

```diff
@@ -3,17 +3,17 @@ import { depositStorage } from './depositStorage';
 
 
 describe('deposit unit tests', () => {
-  const marketAddress = 'test';  
+  const marketId = 'test';  
   const methodName = 'deposit_storage';
   test('deposit storage for one listing', () => {
     
     const args = depositStorage({
       listAmount: 1,
-      marketAddress: marketAddress,
+      marketId: marketId,
     });
       
     expect(args).toEqual({
-      contractAddress: marketAddress,
+      contractAddress: marketId,
```

Comment:
shouldn't this be `contractId` or `contractAccountId`? But I guess this would change just about every single file, so out of scope for this PR